index.js

```diff
@@ -635,6 +635,22 @@
   //. false
   //. ```
 
+  //# Either#extend :: Either a b ~> (b -> b) -> Either a b
+  //.
+  //. Takes a function and returns `this` if `this` is a Left; otherwise it
+  //. returns a Right whose value is the result of applying the function to
+  //. this Right's value. Keep in mind that the difference between this and
+  //. `map` is that the type of the Right remains the same even after
+  //. applying `extend`.
```

Comment:
It's a bit confusing to write _the difference between this and `map`_ in a paragraph with multiple references to `this`. We should write _this function_ or _`extend`_ instead.

Perhaps the second sentence should read:

_`Either#extend` is a restricted form of [`Either#map`][1]: the function provided must return a value of the same type as its input (`Either#map` does not have this restriction)._


[1]: https://github.com/plaid/sanctuary#Either.prototype.map